hyperion/models/post.py

```diff
@@ -39,18 +41,18 @@ class Meta:
     last_modify_date = models.DateTimeField(default=timezone.now)
     content_type = models.CharField(max_length=20, choices=CONTENT_TYPES, default="text/plain")
     visibility = models.CharField(max_length=20, choices=CHOICES, default="PUBLIC")
-    visible_to = models.ManyToManyField(UserProfile, related_name="visible")
+    visible_to = ArrayField(models.CharField(max_length=500), blank=True, default=list)
```

Comment:
Don't you have to revise the `is_accessible` method, since this is changed?

Reply:
Since we are only changing visible_to other visibility are still using userprofile object